dev/io.openliberty.microprofile.telemetry.1.0.internal.jaeger_fat/publish/files/otel-collector-config-zipkin.yaml

```diff
@@ -0,0 +1,24 @@
+receivers:
+ otlp:
+   protocols:
+     grpc:
+     http:
+ zipkin:
+
+exporters:
+ logging:
+ zipkin:
+   endpoint: "http://zipkin-all-in-one:9411/api/v2/spans"
+
+processors:
+ batch:
+
+service:
+  telemetry:
+    logs:
+      level: "debug"
+  pipelines:
+    traces:
+      receivers: [otlp, zipkin]
```

Comment:
I don't think you want the zipkin receiver here. I think the test is expecting to send spans to the collector via OTLP only?